### DISCUSSION TITLE
Recaclculated pole figures

### DISCUSSION THREAD

**Discussion by lilarou** [mtex-toolbox/mtex#1795] [Ask Anything]
Hi MTEX community,

I have pole figure data obtained by XRD measurements.
With the function "odf=calcODF(pf,'silent')" I compute an ODF to then plot recalculated data.
Here is the reconstructed pole figures I get on MTEX
![ODF_75T fas_PDF_RD_TD](https://user-images.githubusercontent.com/123663516/232885277-33dd56c9-31f5-406f-b03a-23f6e5127d7f.jpg)

By loading the same data on ATEX soft and doing ODF reconstruction I get the following pole figures : 
first image is the experimental data
second image is the reconstructed plot
![exp](https://user-images.githubusercontent.com/123663516/232886290-2265013d-b64c-4d8f-a1f0-3a4611976bbc.png)

![recalculated](https://user-images.githubusercontent.com/123663516/232886322-c0b15e87-15ad-4088-8199-2ceb640f9f76.png)

PF are differents from those obtained by MTEX if comparing the (00.2) PF for instance
Do anybody have an idea to explain this difference ? 

Thank you

**Comment by lfgmorales** [comment]
Hi

To me they don’t look too different, you have a maxima in the centre, and some weak girdles on the upper and Lower hemispheres, which on ATEX look broader. Also, the scales you are using in both are totally different, while on mtex you have a maxima of 6 (although your maxima is almost 9), on ATEX you have a maxima of 3.something with broken values


cheers


Luiz

**Reply by lilarou** [reply]
I aggree with you that scales and maxima are differents. What I don't understand is why the girdles on the upper and lower hemispheres have the same maximum pole intensity as the centred pole on ATEX whereas its poles intensity are almost null on MTEX ?

**Comment by ralfHielscher** [comment]
What I do not unterstand is, why ATEX gives you strong peaks in the center of the (100) and (110) pole figure. According to your XRD data there should not be a peak. Maybe ATEX is doing something wrong here?

Ralf.
